packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Providers/NamedTypeSymbolProvider.cs

```diff
@@ -88,12 +154,20 @@ private static CSharpType GetCSharpType(ITypeSymbol typeSymbol)
                         : null);
             }
 
-            var type = System.Type.GetType(fullyQualifiedName);
+            Type? type = System.Type.GetType(fullyQualifiedName);
             if (type is null)
             {
                 throw new InvalidOperationException($"Unable to convert ITypeSymbol: {fullyQualifiedName} to a CSharpType");
             }
```

Comment:
I think we need a TODO item here to support for those types that are purely written in the customization code

Reply:
that is above this line I have a test case for that in the properties already.